internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -83,7 +83,7 @@ var (
 			},
 			ComponentsVersion: ComponentsVersion{
 				KubeletVersion:          "1.15.2",
-				ContainerRuntimeVersion: "1.15.2",
+				ContainerRuntimeVersion: "1.15.0",
```

Comment:
would it be sufficient to just have this version set to `1.15.0` ?

Reply:
@lcavajani not sure I'm following, it's set to `1.15.0`, am I missing something?

Reply:
sorry I mean, would it have worked if we had here `1.15.0` from the beginning ?

Reply:
Yes, it would, but we'd have risked this happening in the future...

Reply:
Thanks, I just wanted to be sure I understand.